bin/fmt

```diff
@@ -1,14 +1,14 @@
-#!/bin/sh
+#!/bin/bash
 
 set -eu
 
 # go list will exclude the vendor directory.  We use grep to exclude generated
 # code.
-dirs=$(go list -f \{\{.Dir\}\} ./... | grep -v controller/gen)
+read -ra dirs <<< "$(go list -f \{\{.Dir\}\} ./... | grep -v controller/gen)"
```

Comment:
Using

`'{{.Dir}}'`

instead of

`\{\{.Dir\}\}`

is perhaps easier to read (and compare with the `go help list` command output), even if the output is of course the same?

Reply:
good suggestions but I'd like to make the minimal change here to fix the script

Reply:
I did elaborate somewhat to make it work with whitespace paths. Shellcheck usually recommends using mapfile for assigning array from command outputs. Using

`mapfile -t dirs < <(go list -f '{{.Dir}}' ./... | grep -v controller/gen)`

Will handle that.